hieradata/development.yaml

```diff
@@ -34,8 +34,6 @@ govuk::node::s_development::apps:
   - 'contacts'
   - 'content_audit_tool'
   - 'content_data_api'
-  - 'content_performance_manager'
-  - 'content_performance_manager::rabbitmq'
```

Comment:
How come we don't `content_data_api::rabbitmq`?

Reply:
That's covered in another Pull Request https://github.com/alphagov/govuk-puppet/pull/9091/files#diff-ff48ae5906656c3a1014d9acb3b0abdcR37